lib/cc/cli/version_checker.rb

```diff
@@ -66,11 +66,13 @@ def http_response
             uri = URI.parse(ENV.fetch("CODECLIMATE_VERSIONS_URL", DEFAULT_VERSIONS_URL))
             uri.query = { version: version, uid: global_config.uuid }.to_query
 
-            request = Net::HTTP::Get.new(uri, "User-Agent" => user_agent)
+            http = Net::HTTP.new(uri.host, uri.port)
+            http.open_timeout = 5
```

Comment:
What led to picking these timeout values? Are there not reasonable automatic defaults? Why are they necessary here but not in `prepare`?